sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1524,6 +1546,15 @@ export class ShareFileClient extends StorageClient {
     uploadResetableStream(streamFactory: (offset: number, count?: number) => NodeJS.ReadableStream, size: number, options?: FileParallelUploadOptions): Promise<void>;
     uploadSeekableBlob(blobFactory: (offset: number, size: number) => Blob, size: number, options?: FileParallelUploadOptions): Promise<void>;
     uploadStream(stream: Readable, size: number, bufferSize: number, maxBuffers: number, options?: FileUploadStreamOptions): Promise<void>;
+    withShareSnapshot(shareSnapshot: string): ShareFileClient;
+}
+
+// @public
+export interface ShareFileRangeList {
```

Comment:
there is `rangeList` property in existing `FileGetRangeListResponse`, not sure whether two different range list names would cause confusions.

Reply:
This `FileGetRangeListResponse` is aliased as `FileGetRangeListDiffResponse`, only used as the response of the `getRangeListDiff`. The original `FileGetRangeListResponse` defined in convenience layer stays unchanged. 
The old `rangeList` is equivalent to the `ranges?: RangeModel[];` here.

```ts
/**
 * Contains response data for the {@link ShareFileClient.getRangeList} operation.
 */
export type FileGetRangeListResponse = FileGetRangeListHeaders & {
  /**
   * Range list for an Azure file.
   *
   * @type {RangeModel[]}
   */
  rangeList: RangeModel[];

  /**
   * The underlying HTTP response.
   */
  _response: HttpResponse & {
    /**
     * The parsed HTTP response headers.
     */
    parsedHeaders: FileGetRangeListHeaders;
    /**
     * The response body as text (string format)
     */
    bodyAsText: string;
    /**
     * The response body as parsed JSON or XML
     */
    parsedBody: RangeModel[];
  };
};
```

Confusion, maybe. Do you have any suggestion?